lib/libc/Kconfig

```diff
@@ -217,6 +217,15 @@ config MINIMAL_LIBC_TIME
 	  In order to make use of the non-reentrant gmtime(), it is necessary
 	  to set CONFIG_MINIMAL_LIBC_NON_REENTRANT_FUNCTIONS=y.
 
+config MINIMAL_LIBC_STRING_ERROR_TABLE
```

Comment:
I'm not sure this configuration entry is useful -- presumably an application using strerror_r expects a useful result. Applications can avoid including this table by avoiding use of this function.

Reply:
I could certainly make it enabled by default.

The main intent is to provide the symbols expected by 3rd-party libraries.

For the sake of minimalism though, if the system designer explicitly chooses to omit the table, then the binary would be smaller, which has some benefit (no need to change application code).